registration/include/pcl/registration/impl/transformation_estimation_lm.hpp

```diff
@@ -258,6 +262,9 @@ pcl::registration::TransformationEstimationLM<PointSource, PointTarget, MatScala
   // Initialize the warp function with the given parameters
   estimator_->warp_point_->setParam (x);
 
+  // Calculate regularization costs
+  double reg_cost = x.dot(estimator_->reg_coeff_ * x);
```

Comment:
2 formulae for the same effect. Why the difference?

Reply:
Good catch. This was just a mistake. I had written both this way, and then decided I liked the other way more (it's been a while now, but I think I just preferred the clearer semantics). I've updated this one to match the prior one.